src/app/events/[slug]/page.tsx

```diff
@@ -35,7 +35,11 @@ export function generateMetadata({ params }: EventProps) {
   const data = getEventData(slug)
 
   return createMetadata({
-    seo: { ...data.seo, image: graphicsData.events1.data.src },
+    seo: {
+      ...data.seo,
+      title: `${data.title} - Event`,
```

Comment:
[Per David's message](https://filecoinfoundation.slack.com/archives/C03S74YDWMC/p1729901387615679)

```suggestion
      title: `seo.data.title || ${data.title} - Filecoin Event`,
```